base/Makefile

```diff
@@ -27,18 +27,20 @@ SHELL := bash
 
 PUSH_TARGET="REMOTE"
 
-REGISTRY:=CONFIGMISSING
-PROJECT:=CONFIGMISSING
-CLUSTER_NAME:=CONFIGMISSING
-ZONE:=CONFIGMISSING
+export REGISTRY:=CONFIGMISSING
```

Comment:
what does this do?

Reply:
it makes it available in the submake. That's how I pass to the challenge Makefile to which registry to push to

Reply:
cool